doc/visualization.dox

```diff
@@ -19,7 +19,7 @@
 @section visualization_Doc Visualization
 
 
-
+  - @ref Doc2dCompImage : Computes and displays image comparisons (squared and absolute differences).
```

Comment:
Compare images and display differences (squared and absolute differences).

?

Reply:
yes better thanks ;)